q2cli/handlers.py

```diff
@@ -101,6 +103,11 @@ def _parse_boolean(self, string):
             ctx = click.get_current_context()
             ctx.exit(1)
 
+    def _add_description(self, option):
+        if self.description != '':
```

Comment:
Based on this comparison, I think `description=''` makes more sense than `description=NoDefault`. The `NoDefault` sentinel is only necessary for default values where `None` is a valid default.

I suppose `description=None` would also work, since `None` is serialized to JSON `null`. I don't have strong feelings about `None` vs `''` though.